### ISSUE TITLE
VSCode 1.29 causes sandbox retrieve/deploy commands to disappear from menu

### ISSUE THREAD

**Issue by DeborahOrth** [forcedotcom/salesforcedx-vscode#748]
### Summary

_Short summary of what is going on or to provide context_.

### Steps To Reproduce:

1.  This is step 1.
1.  This is step 2. All steps should start with '1.'

### Expected result

_Describe what should have happened_.

### Actual result

_Describe what actually happened instead_.

### Additional information

_Feel free to attach a screenshot_.

**VS Code Version**:

**SFDX CLI Version**:

**OS and version**:

**Comment by lcampos** [comment]
Hi @DeborahOrth, this might be a documentation issue. Those commands are displayed if the org you have set as default for the project is a non-scratch org and are hidden if you are working with a scratch org. 

Can you confirm which org type you are working with ?

**Comment by DeborahOrth** [comment]
I only work with non-scratch orgs and I make sure each project I setup has a local default OAuth connection.

**Comment by ntotten** [comment]
We are looking into this and will try to get a resolution shortly.

**Comment by DeborahOrth** [comment]
I tried with 1.29 again but kept getting an Extension host terminated unexpectedly message that gives me options to open Developer Tools or Restart Extension Host.  Restart just leads to the same error so I opened Dev tools console which seemed to indicate that one of my other extensions was having an issue.  I disabled it and reloaded VS Code and now everything is good.  The time to load extensions seems to have slowed down a bit but eventually the delete/deploy/retrieve menu items do show up, including the new line separators.  Thanks for all your help!

**Comment by keirbowden** [comment]
I've created a new project and gone through the same steps as before and now I see the menu items after a couple of seconds, with no errors in the console - previously I'd tried for about 30 minutes or so to get them to show, so I guess something had gone awry around extensions (and restarting with 1.28 handled it - who knows?). 

Thanks for the tip about developer tools - I'll make sure to check that next time I see anything missing.

**Comment by DeborahOrth** [comment]
I agree with Keir, I learned how to use the dev tools with VS Code to help diagnose my own issues, so thanks again.

**Comment by JimSuplizio** [comment]
From @DeborahOrth's comment it looks like another extension (not a Salesforce one?) was causing things to not get loaded. Can you share which one that was?

@keirbowden if you encounter the same thing again and it's another extension can you also share which one it was? It's concerning and maybe VS Code needs to push out a better error message if something doesn't load which can cascade and cause other extensions to not load.

**Comment by DeborahOrth** [comment]
I had problems with the Bracket Pair Colorizer 2 extension.
